simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/handlers/ocpp/CustomerInformationRequestHandler.java

```diff
@@ -0,0 +1,104 @@
+package com.evbox.everon.ocpp.simulator.station.handlers.ocpp;
+
+import com.evbox.everon.ocpp.common.CiString;
+import com.evbox.everon.ocpp.simulator.station.StationMessageSender;
+import com.evbox.everon.ocpp.simulator.station.handlers.ocpp.support.CustomerDataUtils;
+import com.evbox.everon.ocpp.v20.message.common.IdToken;
+import com.evbox.everon.ocpp.v20.message.station.CustomerCertificate;
+import com.evbox.everon.ocpp.v20.message.station.CustomerInformationRequest;
+import com.evbox.everon.ocpp.v20.message.station.CustomerInformationResponse;
+import com.evbox.everon.ocpp.v20.message.station.NotifyCustomerInformationRequest;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Optional;
+
+import static com.evbox.everon.ocpp.common.CiString.CiString512;
+import static com.evbox.everon.ocpp.common.CiString.CiString64;
+import static com.evbox.everon.ocpp.simulator.station.handlers.ocpp.support.CustomerDataUtils.*;
+
+@RequiredArgsConstructor
+public class CustomerInformationRequestHandler implements OcppRequestHandler<CustomerInformationRequest> {
+
+    private static final CiString512 NO_CUSTOMER_DATA_FOUND = new CiString512("No customer data found!");
+    private static final CiString512 CUSTOMER_DATA_CLEARED = new CiString512("Customer data cleared!");
+
+    private StationMessageSender stationMessageSender;
+
+    public CustomerInformationRequestHandler(final StationMessageSender stationMessageSender) {
+        this.stationMessageSender = stationMessageSender;
+        initializeCustomerData();
+    }
+
+    @Override
+    public void handle(String callId, CustomerInformationRequest request) {
+        final var customerIdentifier = Optional.ofNullable(request.getCustomerIdentifier()).map(CiString64::toString).orElse(StringUtils.EMPTY);
```

Comment:
in modern java `StringUtils.EMPTY` is useless you can just use `""`; java use string pools anyway